src/sections/collection/Collection.tsx

```diff
@@ -81,6 +89,12 @@ export function Collection({
                 {t('publishedAlert')}
               </Alert>
             )}
+            {showCollectionFeaturedItems && (
+              <FeaturedItems
+                collectionRepository={collectionRepository}
+                collectionId={collection.id}
+              />
+            )}
```

Comment:
I tested the Collection page, and the user gets Page Not Found if they don't have permission to view the collection.  So is it ok to remove this? I think it would simplify the logic
```suggestion
              <FeaturedItems
                collectionRepository={collectionRepository}
                collectionId={collection.id}
              />
```

Reply:
Great, change applied already.